bpfd-operator/apis/v1alpha1/bpfprogram_types.go

```diff
@@ -22,6 +22,22 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// +genclient
+// +genclient:nonNamespaced
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+//+kubebuilder:resource:scope=Cluster
+
+// BpfProgram is the Schema for the Bpfprograms API
+type BpfProgram struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
Do I need to change this to omitempty?

Reply:
The answer is yes I believe it to be the standard see the pod spec for example -> https://github.com/kubernetes/api/blob/master/core/v1/types.go#L4202